homeassistant/components/hisense_aehw4a1/config_flow.py

```diff
@@ -0,0 +1,46 @@
+"""Config flow for Hisense AEH-W4A1 integration."""
+import logging
+
+from pyaehw4a1.aehw4a1 import AehW4a1
+
+from homeassistant import config_entries
+from homeassistant.helpers import config_entry_flow
+
+from .const import DOMAIN
+from . import CONF_IP_ADDRESS
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class AWSFlowHandler(config_entries.ConfigFlow):
```

Comment:
Please remove the decorator and pass the domain name as domain keyword argument.

https://developers.home-assistant.io/docs/en/config_entries_config_flow_handler.html#defining-your-config-flow